src/views/viewer/ChannelView/ChannelView.style.tsx

```diff
@@ -126,15 +132,93 @@ export const PaginationContainer = styled.div`
 `
 
 export const TabsContainer = styled.div`
+  display: grid;
   margin-bottom: ${sizes(8)};
-  border-bottom: solid 1px ${colors.gray[800]};
-
+  gap: ${sizes(2)};
+  grid-template-rows: 1fr 1fr;
+  grid-template-columns: 1fr 1fr;
+  grid-template-areas:
+    'tabs tabs tabs'
+    'search sort sort';
+  align-items: baseline;
   ${media.compact} {
     padding-top: ${sizes(8)};
   }
+  ${media.small} {
+    border-bottom: solid 1px ${colors.gray[800]};
+    grid-template-areas:
+      'tabs tabs '
+      'search  sort';
+  }
+  ${media.medium} {
+    grid-template-areas: initial;
+    gap: ${sizes(8)};
+    grid-template-rows: 1fr;
+    grid-template-columns: auto 1fr 250px;
+  }
+`
 
+export const SearchContainer = styled.div`
+  display: flex;
+  grid-area: search;
+  width: 100%;
+  align-items: center;
+  ${media.base} {
+    align-self: end;
+  }
+  ${media.small} {
+    align-self: initial;
+  }
   ${media.medium} {
-    display: grid;
-    grid-template-columns: 1fr 250px;
+    grid-area: initial;
   }
 `
+
+export const StyledTabs = styled(Tabs)`
+  grid-area: tabs;
+  ${media.base} {
+    border-bottom: solid 1px ${colors.gray[800]};
+  }
+  ${media.small} {
+    border-bottom: none;
+  }
+  ${media.medium} {
+    grid-area: initial;
+  }
+`
+
+type TextFieldProps = {
+  isOpen?: boolean
+}
+export const StyledTextField = styled(TextField)<TextFieldProps>`
+  transition: all ${transitions.timings.regular} ${transitions.easing};
+  will-change: max-width;
+  width: 100%;
+  align-items: center;
+  max-width: ${({ isOpen }) => (isOpen ? '192px' : '0px')};
+
+  > input {
+    padding: 10px 16px 10px 42px;
+    border: ${({ isOpen }) => (isOpen == false ? 'none !important' : `1px solid ${colors.gray[300]};`)};
+    caret-color: ${colors.blue[500]};
+
+    &:focus {
+      border: 1px solid ${colors.white};
+    }
```

Comment:
Likewise, this likely should be coming from the component?

Reply:
If you look at this prototype of the searchbar it has some on focus styles that we don't have on the base component
https://www.figma.com/proto/TGUpBXgXf9ceMvHbRB5BIx/Joystream-Atlas?page-id=10474%3A305579&node-id=10586%3A292658&viewport=496%2C1603%2C0.4263942539691925&scaling=min-zoom